entities/Base.java

```diff
@@ -0,0 +1,59 @@
+package entities;
+
+import java.util.Date;
+
+public abstract class Base {
+
+    //Attributes
+
+    private Integer id;
+    private String name;
```

Comment:
Remove this property as this entity will be instantiate by all the others entities and not all of then need `name` property.